docs/ftldns/blockingmode.md

```diff
@@ -9,7 +9,7 @@ Pi-hole *FTL*DNS currently supports the following modes for blocking queries:
 Each mode has their advantages and drawbacks which will be discussed in detail below.
 
 !!! note
-    In order to configure a blocking mode, you must edit the *FTL*DNS configuration file (`/etc/pihole/pihole-FTL.conf`). Once you've made any changes to the blocking mode, you must restart Pi-hole with `pihole restartdns`.
+    In order to configure a blocking mode, you must edit the *FTL*DNS configuration file (`/etc/pihole/pihole-FTL.conf`). Once you've made any changes to the blocking mode, you must restart Pi-hole FTL (e.g `sudo service pihole-FTL restart` `sudo systemctl restart pihole-FTL`).
```

Comment:
```suggestion
    In order to configure a blocking mode, you must edit the *FTL*DNS configuration file (`/etc/pihole/pihole.toml`). Once you've made any changes to the blocking mode, you must restart Pi-hole FTL (e.g `sudo service pihole-FTL restart` `sudo systemctl restart pihole-FTL`).
```

Reply:
Already coming in the next commit... :)

Reply:
Happy to wait until you request review :)

Reply:
I figured I would go down a bit of a rabbit hole and try to update some of the outdated sections, too. In two minds about what to do with the `ftldns/configfile/` page... 
 1) Copy paste the current `pihole.toml` into this page as an online reference (with a reference to refer to the actual file for any up to date information
 2) Remove the page entirely
 3) (I know, I said two minds) put a small blurb about the existence of `pihole.toml` and that it is fully commented to describe each section (or to use the web interface)

Thoughts?

Reply:
I would go for three and add a link to `https://ftl.pi-hole.net/master/docs/#get-/config` (currently `development`) if users only want to get a list of possible config options.